crates/store/re_protos/src/v1alpha1/rerun.manifest_registry.v1alpha1.ext.rs

```diff
@@ -266,7 +263,7 @@ pub struct FuzzyComponentDescriptor {
 
 #[derive(Debug, Clone)]
 pub struct QueryLatestAt {
-    pub index: String,
+    pub index: Option<String>,
     pub at: i64,
```

Comment:
I'm not sure what to do with `at` here.

`TimeInt::STATIC` is `TimeInt(None)`, but serialise to `i64::MIN`. The grpc version of `at` was ALREADY `optional` for some reason. Was it meant to be for static though? Any preference on how to go about this?